libstb/secvar/backend/edk2-compat.c

```diff
@@ -102,16 +95,25 @@ static int edk2_compat_process(void)
         /* Check if physical presence is asserted */
         if (is_physical_presence_asserted()) {
                 prlog(PR_INFO, "Physical presence asserted to clear OS Secure boot keys\n");
-                clear_all_os_keys();
+                rc = reset_keystore();
+		if (rc) {
```

Comment:
Inconsistent whitespace, no big deal, will probably do a mass find-and-replace when building the patches